modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -523,12 +523,28 @@ func (suite *KeeperTestSuite) TestChanUpgradeAck() {
 			types.ErrUpgradeNotFound,
 		},
 		{
-			"fails due to proof verification failure, counterparty upgrade connection hops are tampered with",
+			"fails due to upgrade incompatibility",
```

Comment:
tweaked the message slightly because this now hits the `checkForUpgradeCompatibility` call that Cian added recently in Ack.